libraries/stdlib/jvm/src/kotlin/text/CharJVM.kt

```diff
@@ -8,6 +8,8 @@
 
 package kotlin.text
 
+import java.util.*
```

Comment:
Minor: Would more precise import, e.g. `java.util.Locale` be enough?

Reply:
done